main.py

```diff
@@ -30,6 +30,7 @@ def main(args):
     # # print(p.cost())
     #
     # genetic_search(p)
+    bees_alg_opt(p)
```

Comment:
Let's make the main entry point a benchmarking tool for our comparison purposes. Say, add options to choose optimization algorithm, dataset, etc.